plugins/gatsby-ample-debuggers/index.js

```diff
@@ -0,0 +1,3 @@
+import DebugMediaQueries from "./debug-media-queries"
```

Comment:
We should maintain a consistent file structure as much as possible. I would think the media query debugger would go into `src/components`. Ideally every plugin we have is basically a mini Gatsby project (not that it would run on its own).